index.html

```diff
@@ -0,0 +1,1268 @@
+              <h4 class="story-title pt-2">The Newsweek Momentum</h4>
+              <p class="story-short-desc">The Newsweek Momentum Awards will recognize five individuals around the globe, as well as one city.</p>
+            </a>
+          </article>
+        </div>
+
+        <div class="col-12 col-md-3">
+          <article class="post">
+            <a href="#">
+              <div class="post-img-frame">
+              <img src="https://d.newsweek.com/en/full/1515615/janet-yellen-federal-reserve-mnuchin.jpg?w=397&h=265&q=90&f=218421cd54f67d3559360ab61ed9ee5a" alt="some story" class="img-fluid"><span class="post-category">BUSINESS</span></div>
+              <h4 class="story-title pt-2">Ex-Federal Reserve </h4>
+              <p class="story-short-desc">The Newsweek Momentum Awards will recognize five individuals around the globe, as well as one city."</p>
+            </a>
+          </article>
+        </div>
+
+        <div class="col-12 col-md-3">
+          <article class="post">
+            <a href="#">
+              <div class="post-img-frame">
+              <img src="https://d.newsweek.com/en/full/1515677/syria-us-military-coalition-oil.jpg?w=397&h=265&q=90&f=d1dcd882fec8b005b6de46a98598201e" alt="some story" class="img-fluid"><span class="post-category">ANALYSIS</span></div>
+              <h4 class="story-title pt-2">The Newsweek Momentum</h4>
+              <p class="story-short-desc">The Newsweek Momentum Awards will recognize five individuals around the globe, as well as one city.</p>
+            </a>
+          </article>
+        </div>
+
+        <div class="col-12 col-md-3">
+          <article class="post">
+            <a href="#">
+              <div class="post-img-frame">
+              <img src="https://d.newsweek.com/en/full/1511669/spot-im-logo.png?w=397&h=265&q=90&f=fe63931482a6ab2a1825bd5559fc3c72" alt="some story" class="img-fluid"><span class="post-category">EDITOR'S NOTE</span></div>
+              <h4 class="story-title pt-2">To Our Readers:</h4>
+              <p class="story-short-desc">We always want to hear from you, and to make that easier, we've implemented a new commenting forum.</p>
+            </a>
+          </article>
+        </div>
+
+      </div>
+
+    </section>
+
+    <section class="featured-slideshows pt-4">
+      <h6 class="text-center highlights top-title"><img alt="newsweek icon" src="images/newsweekicon.png" class="img-fluid">FEATURED SLIDESHOWS</h6>
+
+      <div class="row">
+        <div class="col-12 col-md-4">
+          <article class="post">
+            <a href="#">
+              <div class="post-img-frame">
+              <img src="https://d.newsweek.com/en/full/1514149/pierre-cardin.jpg?w=397&h=265&l=34&t=27&f=74dc444a96f906b4d71ec24a0df32e52" alt="some story" class="img-fluid"><span class="post-category">13</span></div>
```

Comment:
`SUGGESTION`

Consider making the image `width: 100%`, this can be achieved using the bootstrap class `w-100`. See the difference with the original page below:

|Current|Preffered|
| :-----: | :------: |
|  ![current newsweek fss png](https://user-images.githubusercontent.com/36057474/62257210-56266c80-b3fc-11e9-8c36-3ccc993507d3.png) | ![preffered newsweek fss](https://user-images.githubusercontent.com/36057474/62257213-5b83b700-b3fc-11e9-9cba-abd0a05a13c7.png) |